ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java

```diff
@@ -605,7 +624,7 @@ private RuntimeBuildContext buildContext(
     );
   }
 
-  public static Map<String, Object> updateListProperties(
+  public static void updateListProperties(
```

Comment:
It's confusing to return the same map that's provided as an argument, since it implies we're not modifying the argument map when we in fact are.

Reply:
@vvcephei Thanks for letting us know about this. I think your changes shouldn't break our feature. There are still more changes coming on top of this so don't hold of merging on our account